packages/components/package.json

```diff
@@ -2,13 +2,13 @@
   "name": "@element-plus/components",
   "version": "0.0.5",
   "description": "all components are settled here",
+  "main": "index.ts",
+  "module": "index.ts",
```

Comment:
Shouldn't this be `.js` and `index.mjs`?

Reply:
This file is only formatted , and refactored in a future PR.